ts/components/session/SessionRegistrationView.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { AccentText } from './AccentText';
+
+import { LocalizerType } from '../../types/Util';
+import { RegistrationTabs } from './RegistrationTabs';
+
+declare global {
+  interface Window {
+    displayNameRegex: any;
+  }
+}
+
+interface Props {
+  showSubtitle: boolean;
+  i18n: LocalizerType;
+}
+
+export class SessionRegistrationView extends React.Component<Props> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  public render() {
+    const { showSubtitle, i18n } = this.props;
+
+    return (
+      <div className="session-content">
+        <div id="error" className="collapse" />
+        <div className="session-content-accent">
+          <AccentText showSubtitle={showSubtitle || true} i18n={i18n} />
```

Comment:
This will always evaluate to `true`?

Reply:
yes, wasn't sure if we needed to show the subtitle to I kept it that way. I got my answer: no subtitle. To bad, found it as nice!